content/authentication/troubleshooting-ssh/host-key-verification-failed.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Host key verification failed
+intro: 'As a security precaution, ssh keeps track of which hosts it has previously seen.'
+versions:
+  fpt: '*'
+  ghec: '*'
+topics:
+  - SSH
+shortTitle: Host key verification failed
+---
+
+This means that the key that the server to which your client is connecting presented a key that doesn't match the keys that your client has seen in the past.
+
+It could be because the server changed its keys unexpectedly -- this has happened -- in which case there should be a news reports from trustworthy sources (including the server's official blog) announcing the change.
+
+It could also be that for one reason or another your computer is connecting to a server that is **not** the the right server. In such cases, you probably do not want to connect. You'll probably want to reach out to your support, and then the server's support for help. If the server is being impersonated, the owner of the server will probably appreciate being informed.
```

Comment:
I think we can use more direct language here. "In such cases, you probably do not want to connect" is not necessarily as useful as being able to identify what these cases are.

We use "computer" here but "client" above. Is there any reason for this difference? Again, can we just simplify to "you are connecting"?

```suggestion
If you are encountering the error but can't find an official source for the server's keys, it is safest not to connect, because you may be trying to connect to a server that is not the one you intend. You may want to contact your IT department or the server's support team for help. If the server is being impersonated, the owner of the server will appreciate being informed.
```

Reply:
I think (it was a while ago) I used `computer` because while the client initiated the connection, there are lots of things outside the client that will impact the connection path, and it ends up being the computer that makes the connection to the world. DNS for instance is generally controlled at the computer level, and where the computer physically is does too. 

If that distinction isn't important enough, I'm not terribly attached to it.

Reply:
I'd just go with something like "...you are connecting..." personally. I think it's best to think in terms of what users are doing/trying to do. Talking about the computer connecting to an unintended server sounds quite passive and might even instil panic (like there's something wrong with the computer itself 😁)

Reply:
See what you think and give me a ping when you're ready for me to take another look 🙂

Reply:
Well, there *could* be something wrong with the computer itself! 😄

But, yeah, lemme think about it for a bit. In general, I'm not terribly attached to text -- I prefer incremental improvements even if they aren't my initial wording.

Reply:
I think `being informed` works there! 😁 Because the reader is still the implied agent. I'm not against the passive voice in general, just think it's worth keeping focused on what the reader can/should do. Anyway we might be getting too in the weeds here, so go with whatever you're happy with